### PR TITLE
RELATED: FET-1054 Remove enzyme from sdk-ui-tests

### PR DESCRIPTION
**Pull request by scavnickyj**
---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12152/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14116/ : SUCCESS in 16m 06s

File: libs/sdk-ui-tests/tests/_infra/render.tsx [hunk 1]
```diff
@@ -44,10 +37,12 @@ async function _mountChartAndCapture<T extends VisProps>(
...
     }
 
-    const wrapper = mount(<Component {...(props as any)} />);
+    render(<Component {...(props as any)} />);
+
     const interactions = await promisedInteractions;
 
-    interactions.effectiveProps = effectivePropsExtractor(wrapper);
+    // These are props passed to the top-most component that is being rendered
+    interactions.effectiveProps = props;
```
Comment (kandl) [inline libs/sdk-ui-tests/tests/_infra/render.tsx:45]: 🙌

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dragDropAndMoveWidget.spec.ts**|3(1 skipped)|-|42.3110s|
|✅|**dashboardBasicActions.spec.ts**|2|-|19.1380s|
|✅|**headerSection.spec.ts**|4(1 skipped)|-|76.8230s|
|✅|**dashboardTiger.spec.ts**|9(4 skipped)|-|25.8040s|
|✅|**insightOnDashboard.spec.ts**|6(1 skipped)|-|48.7340s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul/108/).

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3594

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|6.1980s|
|✅|**headlineChartDrilling.spec.ts**|2|-|3.0800s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|12.9160s|
|✅|**attributeFilter.spec.ts**|1|-|6.0220s|
|✅|**barChartDrilling.spec.ts**|4|-|5.8100s|
|✅|**pivotTableSizing.spec.ts**|1|-|4.1710s|
|✅|**dashboardKPI.spec.ts**|8|-|133.1510s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|15.8660s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|138.2740s|
|✅|**dateFilter.spec.ts**|9|-|12.3160s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|6.0200s|
|✅|**dashboard.spec.ts**|9|-|55.8750s|
|✅|**singleSelectionAttributeFilterButton.spec.ts**|3|-|7.8510s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|6.3700s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|18.2800s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|6.3660s|
|✅|**dashboardDateRelated.spec.ts**|8|-|62.3600s|
|✅|**drilling.spec.ts**|7(2 skipped)|-|37.7280s|
|✅|**attributeFilterButton.spec.ts**|13|-|47.9200s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|35.9650s|
|✅|**dashboardDnD.spec.ts**|7|-|57.0160s|
|✅|**dashboardDependentFilter.spec.ts**|8|-|85.1790s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul/491/).

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul/108/ : SUCCESS in 10m 03s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/435/ : FAILURE in 1m 57s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14133/ : FAILURE in 17m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3708/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2552/ : SUCCESS in 11m 06s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul/491/ : SUCCESS in 19m 30s (non-voting)

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3594

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardTiger.spec.ts**|9(4 skipped)|-|24.4670s|
|✅|**dashboardBasicActions.spec.ts**|2|-|14.9170s|
|✅|**insightOnDashboard.spec.ts**|6(1 skipped)|-|41.3400s|
|✅|**dragDropAndMoveWidget.spec.ts**|3(1 skipped)|-|40.6820s|
|✅|**headerSection.spec.ts**|4(1 skipped)|-|61.7880s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul/109/).

**Pull request merged**